package/sbp_log/src/main.c

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (C) 2016 Swift Navigation Inc.
+ * Contact: Gareth McMullin <gareth@swiftnav.com>
+ *
+ * This source is subject to the license found in the file 'LICENSE' which must
+ * be be distributed together with this source. All other rights reserved.
+ *
+ * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+ * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+ */
+
+#include <libsbp/sbp.h>
+#include <libsbp/logging.h>
+#include <czmq.h>
+#include <getopt.h>
+#include <unistd.h>
+
+#define SBP_FRAMING_MAX_PAYLOAD_SIZE 255
+static u16 sender_id = SBP_SENDER_ID;
```

Comment:
sender_id should come from lowest 16 bits of the uuid from the factory data (not hard coded)